src/libcrun/container.c

```diff
@@ -1074,6 +1074,15 @@ container_init_setup (void *args, pid_t own_pid, char *notify_socket, int sync_s
   cleanup_free char *rootfs = NULL;
   int no_new_privs;
 
+#if HAVE_DLOPEN && HAVE_LIBKRUN
+  if (entrypoint_args->context->is_krun == true)
+    {
+      ret = libcrun_configure_libkrun (args, err);
+      if (UNLIKELY (ret < 0))
+        return ret;
+    }
+#endif
+
```

Comment:
let's move this chunk to `libcrun_configure_handler`

Reply:
makes sense.